src/main/resources/application.properties

```diff
@@ -7,3 +7,8 @@ spring.datasource.username=${DB_USERNAME}
 spring.datasource.password=${DB_PASSWORD}
 spring.liquibase.change-log=classpath:/db/changelog/db.changelog-master.yaml
 server.servlet.context-path=/api
+
+logging.level.web=DEBUG
+
+jwt.secret=lskjqlsfjdcsdbgtkedsdfnxdfsdflkhglskglsasfdnauercnalajfcmlahfcmlasfcmwurc
```

Comment:
It's better to store sensitive info in system properties and not pushing it to git.